com.google.Chrome.yaml

```diff
@@ -0,0 +1,94 @@
+app-id: com.google.Chrome
+desktop-file-name-prefix: '(Testing) '
+runtime: org.freedesktop.Platform
+runtime-version: '19.08'
+sdk: org.freedesktop.Sdk
+command: chrome
+separate-locales: false
+build-options:
+  no-debuginfo: true
+finish-args:
+  - '--device=dri'
+  - '--share=ipc'
+  - '--share=network'
+  - '--socket=pulseaudio'
+  - '--socket=x11'
+  - '--require-version=1.8.0'
+  - '--talk-name=org.freedesktop.Notifications'
+  - '--talk-name=org.freedesktop.secrets'
+  - '--talk-name=org.freedesktop.UPower'
+  - '--talk-name=org.gnome.SessionManager'
+  - '--filesystem=home'
```

Comment:
Could you add a comment, explaining why this is needed?

Reply:
It's just the same reason as all the Electron Flatpaks, Chromium doesn't support the XDG portals.

Reply:
Why not just `--filesystem=xdg-download` instead?

Reply:
Hmm, the main idea I had was just that you're generally a lot more likely to download to locations like the Documents folder from your browser than e.g. Discord. That being said, I could instead add the major XDG directories (like documents) here instead of the full home, would that work better?

Reply:
Having the filesystem open is very useful. In fact only the home folder might be too restrictive depending on people's file setups. There are things like the new [Native Filesystem API](https://web.dev/native-file-system/) which users can select whole folders they have access to for say, editing them online.

At *least* all of home should be allowed. However, we may also want to expand this to full access later on should users come forward needing access to things outside of that space.

Reply:
@Garbee (missed this comment previously, sorry) I think it would be easier to expand the set of directories later on in the beta pipeline than restrict it. Recently I was looking through the source, and I think that there *might* actually be a way to force it to use portals, but I'm not sure how that would interact with the native filesystem API; at a glance, it would need probably some more integrated support to regain access to the files after app restart.

At minimum, for the immediate moment, the APIs are still in origin trial, so I think we could get away with a locked down filesystem view at least for the beta phase while the details get sorted out.

Reply:
I would suggest having the home filesystem open for the user since the Native Filesystem API might already be in use and this would cause a lot of finnicky errors.

Reply:
Filesystem access seems to be more debated than I thought :sweat_smile: at minimum like I said, we can probably get away with using this for the beta, and commit to deciding which direction to go in (or maybe if I can make Zypak override to use the native file chooser :eyes:) by an eventual stable release. Native filesystem API is still in origin trials, I don't anticipate this taking *that* long to determine.